ports/tcc-0.9.26/configure

```diff
@@ -29,7 +29,7 @@ mandir=""
 infodir=""
 sysroot=""
 cross_prefix=""
-cc="i386-elf-redox-gcc"
```

Comment:
Still using the cross compiler; look at `ports/tcc.sh` where I added `cross-prefix`.